multimedia/mat2/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                mat2
-version             0.13.4
+version             0.13.5
 revision            1
```

Comment:
`revision` field should be reset to `0` when updating the `version`

Reply:
fixed, sorry this always trips me up...